### DISCUSSION TITLE
Can't disable signups?

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#2088] [Q&A]
Hello,
some days ago installed Vaultwarden to my server. It will be used by me only so I want to disable signups.

Installed it via docker, so:

> nano ~/vaultwarden/docker-compose.yml
> version: '3'
> 
> services:
>   vaultwarden:
>     image: vaultwarden/server:latest
>     container_name: vaultwarden
>     restart: always
>     environment:
>       - WEBSOCKET_ENABLED=true  # Enable WebSocket notifications.
>       - SIGNUPS_ALLOWED=false
>     volumes:
>       - ./vw-data:/data
> 
>   caddy:
>     image: caddy:2
>     container_name: caddy
>     restart: always
>     ports:
>       - 80:80  # Needed for the ACME HTTP-01 challenge.
>       - 443:443
>     volumes:
>       - ./Caddyfile:/etc/caddy/Caddyfile:ro
>       - ./caddy-config:/config
>       - ./caddy-data:/data
>     environment:
>       - DOMAIN=https://my.vaultwarden.com  # Your domain, prefixed with http or https.
>       - EMAIL=my@vaultwarden.com       # The email address to use for ACME registration.
>       - LOG_FILE=/data/access.log

Can you tell me what is bad? After I added - SIGNUPS_ALLOWED=false
I restarted docker container - docker restart vaultwarden.
Then I run docker-compose up -d

Thank you

**Comment by jjlin** [comment]
Disabling signups doesn't disable the button, if that's the issue you're referring to.

See https://github.com/dani-garcia/vaultwarden/wiki/Disable-registration-of-new-users

**Reply by ghost** [reply]
yes but I already added SIGNUPS_ALLOWED=false to docker-compose.yml and nothing changed.

**Reply by jjlin** [reply]
You need to run `docker-compose up -d` afterwards to reload with the new env vars.

**Reply by ghost** [reply]
as I wrote in first post, I already run it

**Reply by jjlin** [reply]
Then you may be overriding it via config.json, see https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview.

**Reply by BlackDex** [reply]
Could you be a bit more clear on what  you mean with `nothing changed`?
What do you expect to be changed? What steps do you take why you think it is not working?

**Comment by Mourad-Tlili** [comment]
It's working just right, the thing is that the 'Create account' button is not being disabled.
Once you try to create and account it wont allow you.
![Capture d’écran 2023-11-23 à 3 45 27 PM](https://github.com/dani-garcia/vaultwarden/assets/46783858/431fa52e-37ee-4966-b026-d44c9278694b)
